src/components/playlist-modal/Modal.css

```diff
@@ -0,0 +1,31 @@
+.playlist-modal {
+	display: block;
```

Comment:
Can remove display: block, below display: flex is overriding this anyway

Reply:
fixed: Removed display: block